lib/bn.js

```diff
@@ -729,6 +930,13 @@ BN.prototype.mul = function mul(num) {
   return this.mulTo(num, out);
 };
 
+// Multiply employing FFT
+BN.prototype.mulf = function mulf(num) {
+  var out = new BN(null);
+  out.words = null;
```

Comment:
@alexeykudinkin I just see it as confusing as it serves no actual purpose, but its existence implies purpose.